tests/php/tests/includes/test_class.wp-job-manager-data-exporter.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * Defines a class to test the data exporter
+ *
+ * @package wp-job-manager-test
+ * @since 1.31.1
+ */
+
+require 'includes/class-wp-job-manager-data-exporter.php';
```

Comment:
We shouldn't need this require in the text class. Instead we should have the Test Class extends WPJM_BaseTest which does some setup of the plugin.